scripts/mppc_to_multiprobe.py

```diff
@@ -0,0 +1,298 @@
+                         "be the similar as in A8 and H8. However, delta y of A1/H1 is {:.2f} "
+                         "whereas delta y of A8/H8 is {:.2f}.".format(delta_y_A1_H1, delta_y_A8_H8))
+
+    # compare A1 with A8: should have same x
+    delta_x_A1_A8 = spotcenter_A1[1] - spotcenter_A8[1]  # pos: CCW rotation, neg: CW rotation
+    # compare H1 with H8: should have same x
+    delta_x_H1_H8 = spotcenter_H1[1] - spotcenter_H8[1]
+
+    # check that values are the same, allow a 10px margin
+    if not almost_equal(delta_x_A1_A8, delta_x_H1_H8, atol=10):
+        raise ValueError("Difference in x position for spots found in mppc cells A1 and A8 should"
+                         "be the similar as in H1 and H8. However, delta x of A1/A8 is {:.2f} "
+                         "whereas delta x of A8/H8 is {:.2f}.".format(delta_x_A1_A8, delta_x_H1_H8))
+
+    # calculate angle for rotation based on center coordinates of the squares
+    # Definition:
+    # CCW rotation: y is positive => need CW rotation output to correct for rotation
+    # CW rotation: x is negative => need CCW rotation output to correct for rotation
+    # the cell images of the input image are stitched together, so need to project all cell images into one (800, 800)
+    # use distances only in the projected image
+    delta_x_A1_H1 = spotcenter_H1[1] - spotcenter_A1[1]
+    angle = numpy.arctan2(delta_y_A1_H1, delta_x_A1_H1)  # [rad]
+    mppc_rotation = -math.degrees(angle)  # convert to degree and change sign for correcting the rotation
+
+    # control
+    delta_x_A8_H8 = spotcenter_H8[1] - spotcenter_A8[1]
+    angle_control = numpy.arctan2(delta_y_A8_H8, delta_x_A8_H8)  # [rad]
+    angle_control_degree = -math.degrees(angle_control)  # convert to degree and change sign for correcting the rotation
+
+    # check that angles are the same, allow a 1 degree margin
+    if not almost_equal(mppc_rotation, angle_control_degree, atol=0.5):
+        raise ValueError("The angle found based on the images of cell A1 and H1, should be the same angle"
+                         "as for cells A8 and H8. However, the angle of A1/H1 is {:.2f} "
+                         "whereas the angle of A8/H8 is {:.2f}.".format(mppc_rotation, angle_control_degree))
+
+    return mppc_rotation  # angle
+
+
+def get_mppc_offset(image, phi):
+    """
+    Calculates the offset of the MPPC detector in space. MPPC detector should be centered in regard to the scanning
+    beamlet, when in single beam mode.
+    Calculates the distance of the active MPPC cell image (bright square) in regard to the center of the cell image.
+    Note: Return value is based on top/left corner (0,0).
+    :param image: (ndarray) Preprocessed E4 cell images.
+    :param phi: (float) Rotation angle to transform the mppc image coordinate system to the multiprobe
+                coordinate system.
+    :returns: (tuple of 2 float) The distance to the E4 cell image center in y and x in px.
```

Comment:
Typically for positions, we use tuples of x,y (and not y,x). Please follow this convention, so that we don't get confused later on.